### PR TITLE
Update philips_js.markdown

### PR DESCRIPTION
**Pull request by snis**
Minor clarifications

## Proposed change

Some minor clarifications, I know thaat this integration is undergoing heavy development.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @snis, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17158).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/philips_js.markdown [hunk 1]
```diff
@@ -13,7 +13,10 @@ ha_config_flow: true
 
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.
 
-Instructions on how to activate the API and if your model is supported can be found [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.
+If your TV respond to http://<ip.of.tv>:1925/system then you are good to go. In the response you should also be able to see which version of the API ("api_version":{"Major":6...) you should use in this integration.
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]:
```suggestion
If your TV responds to `http://IP_ADDRESS_OF_TV:1925/system` then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

File: source/_integrations/philips_js.markdown [hunk 2]
```diff
@@ -13,7 +13,10 @@ ha_config_flow: true
...
+For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.
+
+Also note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]:
```suggestion
Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @snis 👍

**Pull request merged**